collections/_article/dev-snapshot-godot-4-4-dev-6.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Dev snapshot: Godot 4.4 dev 6"
+excerpt: "Featuring: Malware! (does not contain malware)"
+categories: ["pre-release"]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-4-dev-6.webp
+image_caption_title: "Malware"
+image_caption_description: "A game by OddGames"
+date: 2024-12-04 12:00:00
+---
+
+Safety goggles on, people! Features and Quality-of-Life are being integrated at blinding speeds; precautions must be taken to view them head-on. Once secure, we can dive into what's shaping up to be our final development snapshot before 4.4-beta builds roll out!
```

Comment:
Since we'll have dev7 as discussed on RC:
```suggestion
Safety goggles on, people! Features and Quality-of-Life are being integrated at blinding speeds; precautions must be taken to view them head-on. Once secure, we can dive into what's shaping up to be one of the final development snapshots before 4.4-beta builds roll out!
```

Could rephrase further to avoid people misreading it as what it said initially.